src/features/Application/ManageApplicationForm.tsx

```diff
@@ -429,6 +431,33 @@ const ManageApplicationForm: React.FC<IManageApplicationProps> = (props) => {
                 value={fp.values.hacker.application.shortAnswer.skills}
                 showOptionalLabel={true}
               />
+            </div>
+
+            <H1 fontSize={'24px'} marginBottom={'16px'} marginTop={'60px'}>
+              In-person or Remote
```

Comment:
nit: let's name this attendance preference? the description below create's repetition?

Reply:
true, good point